public/reset.css

```diff
@@ -0,0 +1,129 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
This file has nothing to do here ? 
It should be in `src` (and there **is** another copy in `src`!)